server/routes/student.js

```diff
@@ -192,4 +192,44 @@ router.get(
     });
   }
 );
+router.get(
+  '/course/:courseId',
+  check('courseId')
+    .trim()
+    .escape(),
+  checkJWT,
+  async (req, res) => {
+    const user = req.decoded.user;
+    if (req.decoded.user.role === USER_ROLES.STUDENT) {
```

Comment:
(req.decoded.user.role !== USER_ROLES.ADMIN_ACADEMY && req.decoded.user.role !== USER_ROLES.TEACHER)
Anh nghĩ cái này an toàn hơn